Enforce version of maven-resources-plugin from jicofo-deps-2.0.8319.tgz
Index: pom.xml
--- pom.xml.orig
+++ pom.xml
@@ -68,6 +68,11 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>2.6</version>
+      </plugin>
     </plugins>
   </build>
   <modules>
